tools/regression_test.py

```diff
@@ -0,0 +1,686 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import logging
+import argparse
+from pathlib import Path
+from functools import partial
+
+import yaml
+import pandas as pd
+from torch.hub import download_url_to_file
+from torch.multiprocessing import set_start_method
+
+from mmcv.parallel import MMDataParallel
+from mmdeploy.apis import build_task_processor
+from mmdeploy.utils import load_config, get_root_logger, parse_device_id
+from mmdeploy.utils.timer import TimeCounter
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Process Regression Test')
+    parser.add_argument('--deploy-yml', help='regression test yaml path',
+                        default='./configs/mmdet/mmdet_regression_test.yaml')
+    parser.add_argument('--test-type', help='`test type', default='precision')
+    parser.add_argument('--backend', help='test specific backend(s)',
```

Comment:
is this argument used in this version?

Reply:
Done. In my least commit, I make them work.